dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ContentletIndexAPIImpl.java

```diff
@@ -761,13 +763,14 @@ private void addBulkRequest(final BulkIndexWrapper bulk, final List<Contentlet>
 
             final IndiciesInfo info = Sneaky
                     .sneak(() -> APILocator.getIndiciesAPI().loadIndicies());
-            final Gson gson = new Gson(); // todo why do we create a new Gson everytime
             String mapping = null;
 
             try {
 
                 if (this.isWorking(contentlet)) {
-                    mapping = gson.toJson(mappingAPI.toMap(contentlet));
+
+                    mapping = Try.of(()->objectMapper.writeValueAsString(mappingAPI.toMap(contentlet))).getOrElseThrow(
```

Comment:
Probably should encapsulate it into a class (similar concept of the marshall utils)